background/index.ts

```diff
@@ -82,6 +86,79 @@ async function main() {
     },
     ["userOpPool"]
   )
+
+  const fetchData = async () => {
+    const fetchUserOpsSent = async () => {
+      const timeout = 1500
+      console.log(`[background] fetch userOp sent every ${timeout} ms`)
+
+      const s = storage.get()
+      const nm = new NetworkStorageManager(storage)
+
+      const chainId = nm.getActive().chainId
+      const bundler = nm.getActive().bundler
+
+      const userOps = Object.values(s.userOpPool)
+      const sentUserOps = userOps.filter((userOp) => userOp.status === "Sent")
+
+      sentUserOps.forEach(async (sentUserOp) => {
+        const id = sentUserOp.id
+        const userOp = new UserOperation(sentUserOp.userOp)
+        const userOpHash = userOp.hash(sentUserOp.entryPointAddress, chainId)
+        await bundler.wait(userOpHash)
+
+        const userOpReceipt = await bundler.send<{
+          success: boolean
+          reason: string
+          receipt: {
+            transactionHash: HexString
+            blockHash: HexString
+            blockNumber: BigNumberish
+          }
+        }>({
+          method: BundlerRpcMethod.eth_getUserOperationReceipt,
+          params: [userOpHash]
+        })
```

Comment:
Since the context is in the background, I would suggest to add a new function `getUserOperationReceipt` in `BundlerProvider`

Reply:
`BundlerProvider` already has [getUserOperationReceipt()](https://github.com/pilagod/waallet-extension/blob/6023de6f9ee3917c75f398f09d32fa51796a94b7/packages/bundler/provider.ts#L73-L81), but the information it returns is not sufficient to complete `UserOpLog.receipt`. This function should be modified in this PR.